fixtures/eliminator/stealth-wash-zoom.json

```
@@ -57,72 +57,86 @@
         {
           "dmxRange": [6, 10],
           "type": "ColorPreset",
-          "comment": "Red"
+          "comment": "Red",
+          "colors": ["#ff0000"]
         },
         {
           "dmxRange": [11, 15],
           "type": "ColorPreset",
-          "comment": "Green"
+          "comment": "Green",
+          "colors": ["#00ff00"]
         },
         {
           "dmxRange": [16, 20],
           "type": "ColorPreset",
-          "comment": "Blue"
+          "comment": "Blue",
+          "colors": ["#0000ff"]
         },
         {
           "dmxRange": [21, 25],
           "type": "ColorPreset",
-          "comment": "White"
+          "comment": "White",
+          "colors": ["#ffffff"]
         },
         {
           "dmxRange": [26, 30],
           "type": "ColorPreset",
-          "comment": "Red + Green"
+          "comment": "Red + Green aka Yellow",
+          "colors": ["#ffff00"]
         },
         {
           "dmxRange": [31, 35],
           "type": "ColorPreset",
-          "comment": "Green + Blue"
+          "comment": "Green + Blue aka Cyan",
+          "colors": ["#00ffff"]
         },
         {
           "dmxRange": [36, 40],
           "type": "ColorPreset",
-          "comment": "Blue + White"
+          "comment": "Blue + White",
+          "colors": ["#8080ff"]
         },
         {
           "dmxRange": [41, 45],
           "type": "ColorPreset",
-          "comment": "Red + Blue"
+          "comment": "Red + Blue aka Magenta",
+          "colors": ["#ff00ff"]
         },
         {
           "dmxRange": [46, 50],
           "type": "ColorPreset",
-          "comment": "Green + White"
+          "comment": "Green + White",
+          "colors": ["#80c080"]
         },
         {
           "dmxRange": [51, 55],
           "type": "ColorPreset",
-          "comment": "Red + White"
+          "comment": "Red + White",
+          "colors": ["#ff8080"]
         },
         {
           "dmxRange": [56, 60],
           "type": "ColorPreset",
-          "comment": "Red + Green + Blue"
+          "comment": "Red + Green + Blue",
+          "colors": ["#552b55"]
         },
         {
           "dmxRange": [61, 65],
           "type": "ColorPreset",
-          "comment": "Red + Green + White"
+          "comment": "Red + Green + White",
+          "colors": ["#aa8055"]
         },
         {
           "dmxRange": [66, 70],
           "type": "ColorPreset",
-          "comment": "Green + Blue + White"
+          "comment": "Green + Blue + White",
+          "colors": ["#5580aa"]
         },
         {
           "dmxRange": [71, 75],
           "type": "ColorPreset",
-          "comment": "Red + Green + Blue + White"
+          "comment": "Red + Green + Blue + White",
+          "colors": ["#806080"]
```

Red + Green + Blue + White = White

```suggestion
          "colors": ["#ffffff"]
```